### PR TITLE
Update MathJax support

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: helper/header.html [hunk 1]
```diff
@@ -36,9 +36,6 @@
   TeX: { equationNumbers: { autoNumber: "AMS" } }
 });
 </script>
```
Comment (stsievert) [inline helper/header.html:38]: This mathjax config is still valid with the “—mathjax” option, right?

File: readme.md [hunk 2]
```diff
@@ -29,6 +29,7 @@ Even though the docs are *incredibly* useful, I'll include a couple bullets:
 1. It's easiest to clone this repo to get started
 2. All of the content displayed on the pages are in `content/`
 3. Run `cd path/to/jem-press; ruby update.rb` to update your site
+4. Open Google Chrome with `--args --allow-file-access-from-files` to use JQuery `.load()`
```
Comment (stsievert) [inline readme.md]: Can you put in a separate PR for this?

Comment (stsievert) [reply readme.md]: And maybe expand it too. Not everyone will be on Linux and not everyone can open their browser from the command line

**Comment by stsievert** [comment]
Thanks for the PR! 

It looks like most of it is on mathjax. I think I support this if the config is still valid. Using Pandoc support allows easy switching to KaTeX/etc.

File: helper/header.html (hunk 1 above)
Comment (ghost) [reply helper/header.html:38]: Yes. It has been tested using `jem-press/content/latex.md`.
